__tests__/app.test.js

```diff
@@ -78,4 +78,33 @@ describe("GET API/articles/:article_id", () => {
       expect(body.msg).toBe("Bad request")
     })
   })
+})
+describe("Ticket 5.", () => {
+  test("Returned array objects should have each of the specified properties only.", () => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(({body}) => {
+    expect(body.length > 0).toBe(true)
+      body.forEach((article) => {
+        expect(article).toHaveProperty("author")
+        expect(article).toHaveProperty("title")
+        expect(article).toHaveProperty("article_id")
+        expect(article).toHaveProperty("topic")
+        expect(article).toHaveProperty("created_at")
+        expect(article).toHaveProperty("votes")
+        expect(article).toHaveProperty("article_img_url")
+        expect(article).toHaveProperty("comments_count")
+        expect(article).not.toHaveProperty("body")
+      })
+    })
+  })
+  test("200: First item in returned array should be the the least recent article" ,() => {
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then(({body}) => {
+      expect([body[0].created_at, body[1].created_at]).toBeSorted({ descending: true })
```

Comment:
If you wanted to make this a little neater toBeSortedBy allows you to pass an argument of what you want the data to be sorted by - but this works!